index.html

```diff
@@ -170,7 +175,7 @@ <h2>Interactions</h2>
 
             <h4>Using <code>deer-link</code></h4>
             <p class="col-12">In any <code>&lt;deer-view></code> element, the <code>deer-link</code> attribute instructs the templates how to attach links to the rendered object. For lists, each item with an <code>@id</code> property will include an anchor tag hyperlinking
-                to the <code>deer-link</code> URL joined with the <code>@id</code>. This example simply changes the hash on the current page.
+                to the <code>deer-link</code> URL joined with the <code>@id</code>. This example simply changes the hash on the current page. Be sure to be aware of this default behavior if you are using a SPA framework that relies on hashes for routing and consider "Listening to Events" instead.
```

Comment:
Maybe just `Be aware`.  `Be sure to be aware` hits my ears wrong.

Reply:
👂 
🌾 is "Ear of Rice". That's a real thing.